src/please.go

```diff
@@ -991,6 +991,7 @@ func readConfigAndSetRoot(forceUpdate bool) *core.Configuration {
 	}
 	config := readConfig(forceUpdate)
 	// Now apply any flags that override this
+	config.Profiling = opts.Profile != ""
```

Comment:
Perhaps we should set the profile file to some default value (`please.prof`?) if `config.Profile` has been set.

Reply:
Ah, I see, this is a psudo config value that can't be set in the `.plzconfig`. Think this makes sense.